package.json

```diff
@@ -55,28 +54,28 @@
   "devDependencies": {
     "autoprefixer": "6.7.7",
     "babel-core": "6.26.0",
-    "babel-eslint": "8.0.1",
+    "babel-eslint": "8.0.3",
     "babel-jest": "21.2.0",
-    "babel-loader": "6.4.1",
+    "babel-loader": "7.1.2",
     "babel-plugin-lodash": "3.2.11",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-plugin-transform-react-jsx": "6.24.1",
     "babel-plugin-transform-runtime": "6.23.0",
     "babel-preset-env": "1.6.0",
     "babel-traverse": "6.26.0",
     "check-node-version": "3.1.1",
-    "codecov": "2.3.0",
+    "codecov": "3.0.0",
```

Comment:
Are we still using this?

Reply:
Yes, we display badge in Readme based on it.